website/docs/docs/deploy/ci-jobs.md

```diff
@@ -91,10 +91,20 @@ The green checkmark means the dbt build and tests were successful. Clicking on t
 
 If you're experiencing any issues, review some of the common questions and answers below.
 
+<details>
+  <summary>Temporary schemas aren't dropping</summary>
+  <div>
+    <div>If your temporary schemas aren't dropping after a PR merges or closes, this typically indicates you have overridden the <code>generate_schema_name</code> macro and it isn't using <code>dbt_cloud_pr_</code> as the prefix. To resolve this, change your macro so that the temporary PR schema name contains the required prefix. For example: 
```

Comment:
sugg line break to make it easier to read
```suggestion
    <div>If your temporary schemas aren't dropping after a PR merges or closes, this typically indicates you have overridden the <code>generate_schema_name</code> macro and it isn't using <code>dbt_cloud_pr_</code> as the prefix.<br></br><br></br> To resolve this, change your macro so that the temporary PR schema name contains the required prefix. For example: 
```

Reply:
currently looking like this: 
![Screenshot 2023-09-19 at 11 10 35](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/07e09660-efe0-4d85-afeb-614706948d54)

Reply:
this was a copy pasta of what was there before. happy to add!